ext/standard/mail.c

```diff
@@ -281,6 +320,11 @@ PHPAPI int php_mail(char *to, char *subject, char *message, char *headers, char
 		efree(f);
 	}
 
+	if (hdr && php_mail_detect_multiple_crlf(hdr)) {
+		php_error_docref(NULL TSRMLS_CC, E_WARNING, "Multiple or malformed newlines found in additional_header");
+		MAIL_RET(0);
```

Comment:
@yohgaki mail headers are use to send attachments, I don't see how this can be seen as misuse. This is a valid use case, which should not be excluded. There is no other way to send attachments in PHP.